common/tier4_tag_utils/include/tier4_tag_utils/lidartag_hypothesis.hpp

```diff
@@ -48,19 +47,17 @@ class LidartagHypothesis
   cv::Point3d getCenter() const;
 
   double getTransCov() const;
-  double getTransDotCov() const;
   double getRotCov() const;
-  double getRotDotCov() const;
   double getSpeed() const;
 
   bool converged() const;
   double timeSinceFirstObservation(const rclcpp::Time & stamp) const;
   double timeSinceLastObservation(const rclcpp::Time & stamp) const;
 
-  void setDynamicsModel(DynamicsModel dynamics_mode);
   void setMinConvergenceTime(double convergence_time);
   void setMaxNoObservationTime(double time);
-  void setMaxConvergenceThreshold(double transl, double tansl_dot, double angle, double angle_dot);
+  void setMaxConvergenceThreshold(
+    double transl, double translation_dot, double angle, double angle_dot);
```

Comment:
Changed in [a23cea2](https://github.com/tier4/CalibrationTools/pull/148/commits/a23cea234f232fbd7c9b4a31c2976518f4e83918)
This is a little bit of a delicate change as it can break several things that can not be checked by compiling. Please review accordingly